README.md

```diff
@@ -6,6 +6,7 @@ This repository contains Bazel rules for generating Google Cloud client
 libraries using the corresponding generators:
 
 - [C#](https://github.com/googleapis/gapic-generator-csharp)
+- [Go](https://github.com/googleapis/gapic-generator-go)
```

Comment:
The rules in here aren't actually used by gapic-generator-go anymore (we need to clean these up), so please revert this.